RallyHereIntegration/Source/RallyHereIntegration/Private/RH_GameInstanceBootstrappers.cpp

```diff
@@ -136,13 +141,24 @@ void URH_GameInstanceServerBootstrapper::Initialize()
 			, *RH_GETENUMSTRING("/Script/RallyHereIntegration", "ERH_ServerBootstrapMode", BootstrapMode));
 	}
 
-	if (FParse::Value(FCommandLine::Get(), TEXT("rhmaxrecyclecount="), MaxRecycleCount))
+	if (FParse::Value(FCommandLine::Get(), TEXT("rh.maxrecyclecount="), MaxRecycleCount))
```

Comment:
Is this an intentional breaking change? Should we support both?

Reply:
As above, it was to make things more consistent with our other non-GHA parameters, but I am not sold on it tbh. Will remove it for now as this PR has a bunch of more important things.